### PR TITLE
 first_commit

### PR DESCRIPTION
**Pull request by eugsim1**
snmp okv utilty

### CODE REVIEW

File: manageability-and-operations/observability-and-manageability/okv-monitoring/files/main.go [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+	params := &g.GoSNMP{
+		Target:        "10.0.0.190",
+		Port:          161,
+		Version:       g.Version3,
+		SecurityModel: g.UserSecurityModel,
+		MsgFlags:      g.AuthPriv,
+		Timeout:       time.Duration(30) * time.Second,
+		SecurityParameters: &g.UsmSecurityParameters{UserName: *UserNamePtr,
+			AuthenticationProtocol:   g.SHA,
+			AuthenticationPassphrase: "Welcome123.",
```
Comment (oheimburger) [inline manageability-and-operations/observability-and-manageability/okv-monitoring/files/main.go:73]: No hardcoded passwords!

File: manageability-and-operations/observability-and-manageability/okv-monitoring/files/main.go [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+			PrivacyProtocol:          g.AES,
+			PrivacyPassphrase:        "Welcome123.",
```
Comment (oheimburger) [inline manageability-and-operations/observability-and-manageability/okv-monitoring/files/main.go:75]: No hardcoded passwords!

**Review by oheimburger — CHANGES_REQUESTED** [review]
Remove the branch, hardcoded passwords are not allowed.

**Review by eugsim1 — COMMENTED** [review]
done

**Comment by eugsim1** [comment]
modification is done

**Review by eugsim1 — COMMENTED** [review]
please review the new commit with some update on monitoring scripts as well

**Pull request closed without merging**